index.d.ts

```diff
@@ -8,13 +10,17 @@ declare namespace Cypress {
     composed?: Boolean;
   };
 
+  type CommandOptions = {
```

Comment:
It appears `CommandOptions` is [already declared as an interface](https://github.com/cypress-io/cypress/blob/9aeadbdfa20e1824dc0150b79182516ec97c6af8/cli/types/index.d.ts#L45) in Cypress library.

As such a typescript error is issued because you can't redeclare a type. Can we change this type name to `ShadowOptions` or something similar?  If we extend `CommandOptions` by declaring this an interface, it results in `prevSubject` being required.

Reply:
I know this was awhile ago, but I'm having this exact problem. I'm using Typescript with Cypress and I am unable to specify a timeout on `shadowGet()` or `shadowFind()` because {  timeout: 5000 } is not assignable to parameter of type `Cypress.CommandOptions`.  Any chance this is something we could fix? I can open a new issue if necessary.

Reply:
I see these typings were not updated properly:
https://github.com/abramenal/cypress-shadow-dom/blob/master/index.d.ts#L25

Are you able to create a MR with that fix?

Reply:
Is it as simple as changing the `shadowFind`, `shadowFirst`, `shadowGet` and `shadowLast` methods to use `ShadowCommandOptions`? Also looks like you have a typo in `ShadowEventOptios` - I could fix that too.